src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -838,31 +894,48 @@ def generate_FDB_5_default_config(self, filename):
     device_b = json.load(
         open(os.path.join('testcases', 'testdata', 'device_b.json')))
 
+    # FSS database test harness configuration
+    # The database File FDB_5_default_allsitedata.json has one FSS site information.   
+    fss_database_config = {
+        'hostName': 'localhost',
+        'port': 8000,
+        'fileUrl': '/db_sync',
+        'filePath': os.path.join('testcases', 'testdata', 'fdb_5', 'FDB_5_default_allsitedata.json')
+    }
+
+    # GWBL database test harness configuration.
+    # The GWBL database file l_micro.zip has one GWBL information within 150 KMs
```

Comment:
Can you mention which GWBL specifically? This will make validation easier.

Reply:
FDB.5     Mentioned the GWBLs along with the "Unique System Identifier"   within 150 km from FSS sites